mlflow/models/model.py

```diff
@@ -68,6 +68,26 @@ def get_input_schema(self):
     def get_output_schema(self):
         return self.signature.outputs if self.signature is not None else None
 
+    def load_input_example(self, path: str):
+        """
+        Load the input example saved along a model. Returns None if there is no example metadata
+        (i.e. the model was saved without example). Raises FileNotFoundError if there is model
+        metadata but the example file is missing.
+
+        :param path: Path to the model directory.
+
+        :return: Input example (NumPy ndarray, SciPy csc_matrix, SciPy csr_matrix,
+                 pandas DataFrame, dict) or None if the model has no example.
+        """
+
+        # Just-in-time import to only load example-parsing libraries (e.g. numpy, pandas, etc.) if
+        # example is requested.
+        from mlflow.models.utils import (
+            _read_example,
+        )
```

Comment:
Not a big issue, but we can remove the comma. 

```suggestion
        from mlflow.models.utils import _read_example
```

Reply:
Reformatted to remove the comma